packages/patternfly-4/react-core/src/components/Pagination/Pagination.md

```diff
@@ -87,6 +87,48 @@ class PaginationBottom extends React.Component {
 }
 ```
 
+## Pagination disabled
+```js
+import React from 'react';
+import { Pagination, PaginationVariant } from '@patternfly/react-core';
+
+class PaginationTop extends React.Component {
```

Comment:
nit: class name should be `PaginationDisabled`. Doesn't affect the rendering but shows up in the sample code